vernac/comInductive.mli

```diff
@@ -59,7 +59,3 @@ val template_polymorphism_candidate :
    can be made parametric in its conclusion sort, if one is given.
    If the [Template Check] flag is false we just check that the conclusion sort
    is not small. *)
-
-val sign_level : Environ.env -> Evd.evar_map -> Constr.rel_declaration list -> Univ.Universe.t
-(** [sign_level env sigma ctx] computes the universe level of the context [ctx]
-    as the [sup] of its individual assumptions, which should be well-typed in [env] and [sigma] *)
```

Comment:
Is this OK wrt https://github.com/coq/coq/pull/10647 ?

Reply:
cc @Janno

Reply:
Since my PR only splits up one function into two I don't see any reason why it wouldn't work anymore. Given my limited time right now we should probably merge this change first and I'll worry about mine later.

Reply:
I think @SkySkimmer was asking if you are using the `sign_level` function.

By the way @Janno you should try to update your PR soon if you want it to make into 8.11, the freeze should happen soon.